features/bootstrap/FeatureContext.php

```diff
@@ -32,7 +33,11 @@ class FeatureContext implements Context
      */
     public function __construct()
     {
-        (new Dotenv())->load(dirname(dirname(__DIR__)) . '/.env.behat');
+        try {
+            (new Dotenv())->load(dirname(dirname(__DIR__)) . '/.env.behat');
+        } catch (PathException $exception) {
+            // Prevent non-existent file exception while CI execution
```

Comment:
Ajouter:  `print "The .env.behat file does not exist. This is probably unintentional."`